### ISSUE TITLE
bug in ycbcr_to_rgb function

### ISSUE THREAD

**Issue by fenneishi** [kornia/kornia#2348]
### Describe the bug

https://github.com/kornia/kornia/blob/2c084f8dc108b3f0f3c8983ac3f25bf88638d01a/kornia/color/ycbcr.py#L70
#### now:
 return torch.stack([r, g, b], -3)
#### need to be:
 return torch.stack([r, g, b], -3).clamp(0,1)
#### because:
![image](https://user-images.githubusercontent.com/33345538/234479794-79174b56-4041-4e9c-96f3-d3117ce44104.png)

**Comment by edgarriba** [comment]
@fenneishi clamp or norm_min_max 0/1 ? @cceyda any thoughts here ?

**Comment by fenneishi** [comment]
yes, you need clamp the return of ycbcr_to_rgb function into [0,1].
Because you may get "cb=1,cr=0" as ithe input of this function.
This input(cb=1,cr=0) may from a sigmoid function which is  the last layer of a neural network

**Comment by edgarriba** [comment]
i see -- feel free to send a pull request with the fix and tests

**Comment by juliaaz** [comment]
@edgarriba Hi Edgar! I opened a pull request for this issue, however it looks like the 2-3 tests are failing because of the code of other user related to Ransac, which is irrelevant in case of this bug. Any thoughts on how to move on?

**Comment by johnnv1** [comment]
> @edgarriba Hi Edgar! I opened a pull request for this issue, however it looks like the 2-3 tests are failing because of the code of other user related to Ransac, which is irrelevant in case of this bug. Any thoughts on how to move on?

Don't worry, they are unrelated with your PR
